routing/filter_creation_metrics.go

```diff
@@ -0,0 +1,103 @@
+package routing
+
+import (
+	"time"
+
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	maxAge              = 2
+	metricsPrefix       = "routeCreationTime."
+	defaultCreationTime = "default"
+)
+
+type FilterCreationMetrics struct {
```

Comment:
As we discussed, the notion of "filter creation" is misleading because it is not the filter whose creation we can track but the route. This naming should be changed across the entire PR.